controllers/edgedevice_controller.go

```diff
@@ -48,6 +48,7 @@ type EdgeDeviceReconciler struct {
 //+kubebuilder:rbac:groups=objectbucket.io,resources=objectbucketclaims,verbs=get;list;watch;create;update;patch
 //+kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch
 //+kubebuilder:rbac:groups=core,resources=secrets,verbs=get;list;watch
+//+kubebuilder:rbac:groups=core,resources=events,verbs=create;patch
```

Comment:
Running `make manifests` introduces changes to roles.yaml file; please regenerate it and submit as part of this PR.

Reply:
No need, the roles.yaml was updated without adding annotation for it in a previous PR. This PR aligns the state of the roles.yaml by adding the needed annotation.

If we run make manifests against the main branch, without any changes, we'll end up with
```
→ git diff
diff --git a/config/rbac/role.yaml b/config/rbac/role.yaml
index ac2a5e3..c294c17 100644
--- a/config/rbac/role.yaml
+++ b/config/rbac/role.yaml
@@ -6,13 +6,6 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
-  - patch
-  - create
 - apiGroups:
   - ""
   resources:
```

With this PRs no changes to that file.

Reply:
Which means that even if what we want is in the file, it is not consistent with what would be generated. 
If someone makes other changes and runs `make manifests` it will show changes around `events` resource, which will probably have nothing to do with that PR.

Reply:
This is already part of the codebase. The part of `config/rbac/role.yaml` was already added by https://github.com/jakub-dzon/k4e-operator/commit/428f41aa1a3f27fa2e4e1bfac1af4814e8796c26#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7R9

Reply:
Because I've added it manually, wasn't ever of this auto-generation.

Reply:
It is a part of the codebase, but re-generation rearranges the contents of the file; let's clean it up here.

Reply:
@jakub-dzon sorry for the confusion. I see your point now :-)
Added the regenerated file.